homeassistant/components/light/blinksticklight.py

```diff
@@ -0,0 +1,64 @@
+"""
+homeassistant.components.light.blinkstick
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Blinkstick lights.
+"""
+
+from blinkstick import blinkstick
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+from homeassistant.components.light import (Light, ATTR_RGB_COLOR)
+
+REQUIREMENTS = [ "blinkstick==1.1.7" ]
+DEPENDENCIES = []
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+	stick = blinkstick.find_by_serial( config['serial'])
+
+	add_devices_callback( [BlinkStickLight(stick, config['name'])] )
+
+class BlinkStickLight(Light):
+	def __init__( self, stick, name ):
+		self._stick = stick
+		self._name = name
+		self._serial = stick.get_serial()
+		r,g,b = stick.get_color()
+		self._rgb_color = [r, g, b]
+
+	@property
+	def should_poll(self):
```

Comment:
I think I assumed that polling was unnecessary since the state will only change if HA commands it, and update_ha_state is called at that point.